systemtest/src/main/java/io/strimzi/systemtest/resources/minio/SetupMinio.java

```diff
@@ -182,4 +182,21 @@ public static void waitForDataInMinio(String namespace, String bucketName) {
             return bucketSize > 0;
         });
     }
+
+    /**
+     * Wait until size of the bucket is 0 B.
+     * @param namespace Minio location
+     * @param bucketName bucket name
+     */
+    public static void waitForNoDataInMinio(String namespace, String bucketName) {
```

Comment:
I guess it's not for this PR and it's just an idea, but if the class is named `SetupMinio`, I would not expect the methods that should be in some "utils" class. (but I approved the PR where it was originally added, it just came to my mind now).

Reply:
I can move it or rename `SetupMinio` -> `Minio`, what do you prefer?